src/main/resources/messages_en.properties

```diff
@@ -904,4 +904,9 @@ program.name.empty=Program name can not contain only blank spaces. Please check
 program.name.max.length.exceeded=Program name can not have more than 255 characters. Please check 
 program.name.invalid.characters=Program name can not contain the following characters: /\":*?<>| 
 program.start.date.invalid=Program start date has an invalid format. Valid format is yyyy-mm-dd. Please check 
-program.name.already.exists=A program with name {0} for the crop: {1} was found in the system. Please check
\ No newline at end of file
+program.name.already.exists=A program with name {0} for the crop: {1} was found in the system. Please check
+
+germplasm.merge.cannot.merge.germplasm.with.progeny=Some of the germplasm to be deleted has progeny associated to it. Please manually edit such progeny so that it points to a different progenitor in order to proceed with the merge.
```

Comment:
For failed validations (has progeny, fixed) - I think it's better to show which GIDs failed so they know which to edit